core/src/main/java/edu/wpi/grip/core/operations/Operations.java

```diff
@@ -20,82 +20,86 @@
-                PublishVideoOperation::new,
-                DistanceTransformOperation::new,
-                NormalizeOperation::new,
-                WatershedOperation::new,
-                SwitchOperation::new,
-                ValveOperation::new,
-                ThresholdMoving::new
+                // Composite operations
+                new OperationMetaData(BlurOperation.DESCRIPTION, () -> new BlurOperation(isf, osf)),
+                new OperationMetaData(ConvexHullsOperation.DESCRIPTION, () -> new ConvexHullsOperation(isf, osf)),
+                new OperationMetaData(DesaturateOperation.DESCRIPTION, () -> new DesaturateOperation(isf, osf)),
+                new OperationMetaData(DistanceTransformOperation.DESCRIPTION, () -> new DistanceTransformOperation(isf, osf)),
+                new OperationMetaData(FilterContoursOperation.DESCRIPTION, () -> new FilterContoursOperation(isf, osf)),
+                new OperationMetaData(FilterLinesOperation.DESCRIPTION, () -> new FilterLinesOperation(isf, osf)),
+                new OperationMetaData(FindBlobsOperation.DESCRIPTION, () -> new FindBlobsOperation(isf, osf)),
+                new OperationMetaData(FindContoursOperation.DESCRIPTION, () -> new FindContoursOperation(isf, osf)),
+                new OperationMetaData(FindLinesOperation.DESCRIPTION, () -> new FindLinesOperation(isf, osf)),
+                new OperationMetaData(HSLThresholdOperation.DESCRIPTION, () -> new HSLThresholdOperation(isf, osf)),
+                new OperationMetaData(HSVThresholdOperation.DESCRIPTION, () -> new HSVThresholdOperation(isf, osf)),
+                new OperationMetaData(MaskOperation.DESCRIPTION, () -> new MaskOperation(isf, osf)),
+                new OperationMetaData(NormalizeOperation.DESCRIPTION, () -> new NormalizeOperation(isf, osf)),
+                new OperationMetaData(PublishVideoOperation.DESCRIPTION, () -> new PublishVideoOperation(isf, osf)),
+                new OperationMetaData(ResizeOperation.DESCRIPTION, () -> new ResizeOperation(isf, osf)),
+                new OperationMetaData(RGBThresholdOperation.DESCRIPTION, () -> new RGBThresholdOperation(isf, osf)),
+                new OperationMetaData(SwitchOperation.DESCRIPTION, () -> new SwitchOperation(isf, osf)),
+                new OperationMetaData(ValveOperation.DESCRIPTION, () -> new ValveOperation(isf, osf)),
+                new OperationMetaData(WatershedOperation.DESCRIPTION, () -> new WatershedOperation(isf, osf)),
+
+                // OpenCV operations
+                new OperationMetaData(MatFieldAccessor.DESCRIPTION, () -> new MatFieldAccessor(isf, osf)),
+                new OperationMetaData(MinMaxLoc.DESCRIPTION, () -> new MinMaxLoc(isf, osf)),
+                new OperationMetaData(NewPointOperation.DESCRIPTION, () -> new NewPointOperation(isf, osf)),
+                new OperationMetaData(NewSizeOperation.DESCRIPTION, () -> new NewSizeOperation(isf, osf)),
+
+                // Raw OpenCV operations
```

Comment:
Remove this comment and associated newlines since OpenCV operations are in their own file